### ISSUE TITLE
Long tab titles are truncated.

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#113]
Originally reported on Google Code with ID 111

```
What steps will reproduce the problem?
1. Try to easily see the full title of a long tab.

What is the expected output? What do you see instead?
It's too difficult. Should be easy.

Please provide any additional information below.
I figure I can put an option in the Tabs category that would allow for tab titles to
wrap.

I'm forking this from issue 97, since that veered off into a different direction.
```

Reported by `pcxunlimited` on 2010-12-13 01:10:52

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 22:16:41
- **Labels added**: Milestone-5

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:04:23
- **Labels added**: Type-External
- **Labels removed**: Type-Bug

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:08:06
- **Labels added**: Type-Internal
- **Labels removed**: Type-External

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:17:41
- **Labels removed**: OS-All

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:39:18
- **Labels removed**: Priority-Low

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:47:28
- **Status changed**: `Assigned`
